### PR TITLE
Support custom list of colors in `draw_tracked_boxes` (renames `line_color` to `border_colors` and `line_width` to `border_width`).

### PR DESCRIPTION
**Pull request by aguscas**

### CODE REVIEW

File: norfair/drawing.py [hunk 1]
```diff
@@ -223,7 +223,7 @@ def draw_boxes(frame, detections, line_color=None, line_width=None, random_color
 def draw_tracked_boxes(
     frame,
     objects,
-    line_color=None,
+    line_color=None,
```
Comment (dekked) [inline norfair/drawing.py:226]: The name of the parameter doesn't convey that it can be a tuple. It's confusing.

Maybe name it `line_colors` or just `colors`?

Or maybe renaming both to `border_colors` and `border_width` makes more sense?

File: norfair/drawing.py [hunk 2]
```diff
@@ -223,7 +223,7 @@ def draw_boxes(frame, detections, line_color=None, line_width=None, random_color
 def draw_tracked_boxes(
```
Comment (dekked) [inline norfair/drawing.py:223]: This function is in need of some documentation, although maybe not for this PR.

File: norfair/tracker.py [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import math
 from typing import Callable, List, Optional, Sequence
+from copy import deepcopy
```
Comment (dekked) [inline norfair/tracker.py:3]: These changes are unrelated and should not be on this PR. Please rebase interactive to remove them :)

**Review by dekked — CHANGES_REQUESTED** [review]

File: norfair/drawing.py [hunk 4]
```diff
@@ -223,25 +223,29 @@ def draw_boxes(frame, detections, line_color=None, line_width=None, random_color
...
+    if border_width is None:
+        border_width = int(frame_scale * 0.5)
     if id_size is None:
         id_size = frame_scale / 10
     if id_thickness is None:
         id_thickness = int(frame_scale / 5)
-    color_is_None = line_color == None
-    for obj in objects:
+    if isinstance(border_colors, tuple):
+        border_colors = [border_colors]
```
Comment (joaqo) [inline norfair/drawing.py:240]: Is there a reason tuples have to be converted to list?

**Review by joaqo — COMMENTED** [review]
Looks good to me. PRs should contain a short description though, makes it easier to review them.
